extras.py

```diff
@@ -37,20 +37,17 @@ def amt_filter(inp):
     return '''{}Bot is up. Configuration:
```

Comment:
There were supposed to be changes for this line, I will need to push a new commit